proto/topology.proto

```diff
@@ -98,7 +98,7 @@ message ZoneData {
 
 message ServerData {
     required uint32 serverId = 1;
-    required string hostName = 2;
+    optional string hostName = 2;
```

Comment:
Since you have changed this field into optional, do the corresponding codes need to be modified？

Reply:
I think I have changed all the corresponding function. I will check it again.